.github/workflows/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+## Description
```

Comment:
Maybe here we can use the same template that is used for swapr-dapp, it would be more consistent overall I guess, what do you think?

[template](https://github.com/levelkdev/swapr-dapp/blob/develop/.github/pull_request_template.md)